simulation/tests/test_agents.py

```diff
@@ -848,15 +852,17 @@ def test_landlord_heat_pump_hassle_if_not_already_installed(
             or occupant_type == OccupantType.RENTED_SOCIAL
         ):
             assert (
-                household.reset_heating_system_hassle(heating_system_hassle_factor=0.1)
+                household.reset_heating_system_hassle(
```

Comment:
Could make this test a lot clearer by:
```
rented_hassle_factor = 0.4
owner_occupier_hassle_factor = 0.1

assert household.reset_heating_system_hassle(
     heating_system_hassle_factor=owner_occuper_hassle_factor, 
     rented_heating_system_hasstle_factor=rented_hassle_factor
) == rented_hassle_factor
```

Try to avoid arbitrary hard coded constants without context